src/shared/style/style.css

```
@@ -0,0 +1,200 @@
+* {
+  font-family: Pretendard, sans-serif;
+}
```

💊 제안  
이렇게 전체선택자로 모든 태그에 font-family 값을 Pretendard로 주시는 것보다 최상위 태그에 해당 속성을 명시하고 자식 태그들은 상속받게 하는 것이 중복을 줄일 수 있고 가독성 측면에서도 더 좋을 것 같습니다~


```suggestion
html {
  font-family: Pretendard, sans-serif;
}
```